libdnf/CMakeLists.txt

```diff
@@ -62,8 +62,12 @@ include_directories(${GLIB2_INCLUDE_DIRS})
 target_link_libraries(libdnf ${GLIB2_LIBRARIES})
 
 # GPGME
-PKG_CHECK_MODULES(GPGME gpgme REQUIRED)
-list(APPEND LIBDNF_PC_REQUIRES "${LIBREPO_MODULE_NAME}")
+pkg_check_modules(GPGME gpgme)
+if(NOT GPGME_FOUND)
+    set(GPGME_MODULE_NAME gpgme)
+    set(GPGME_LIBRARIES gpgme)
+endif()
+list(APPEND LIBDNF_PC_REQUIRES "${GPGME_MODULE_NAME}")
```

Comment:
Could you add an explanatory comment saying this `if` is for CentOS Stream? Also, (unless you've already done so), consider reporting the missing pkgconfig somewhere in Stream?

Reply:
The pc files were removed instead of editing them (which would have been the correct fix):

```
* Mon Jun 01 2020 Jiri Kucera - 1.13.1-2
  - Remove *.pc files due to 'nothing provides pkgconfig(gpg-error)'
    and 'nothing provides pkgconfig(libassuan)' dnf errors
    Related: #1829822
```

The wrong fix was done in gpgme, and that should be corrected. This has terrible knock-on effects for others too.

Reply:
A commit explaining the `if` added.
I've filed a RHEL 8 bug to bring them back: https://bugzilla.redhat.com/show_bug.cgi?id=1953905